roles/nfs/common/tasks/check.yml

```diff
@@ -118,3 +118,20 @@ Please define the CES shared root file system mount point in the inventory."
    that:
    - ccr_status.rc == 0
    fail_msg: "CCR is not enabled on the cluster. CCR must be enabled to install protocols"
+
+- name: Install python netaddr library
```

Comment:
We can't install through pip.  we need to use  yum, zypper or apt module based on os. sorry i forgot to add this in my review comment.

Reply:
Can we add when condition here as well , this should happen only if ipv6 is defined